src/components/registerpage/Register/__snapshots__/index.test.js.snap

```diff
@@ -0,0 +1,205 @@
+
+exports[`Register Component matches the tree 1`] = `
+<form
+  noValidate={true}
+  onSubmit={[Function]}
+>
+  <div
+    className="loginContainer"
+  >
+    <div
+      className="loginContainer__loginTitle"
+    >
+      register.create
+    </div>
+    <div
+      className="loginContainer__loginDialog"
+    >
+      <div
+        className="loginContainer__loginDialog__inputLabel"
+      >
+        register.name
+      </div>
+      <div
+        className="loginContainer__loginDialog__input"
+      >
+        <input
+          className="ant-input"
+          onBlur={[Function]}
+          onChange={[Function]}
+          onFocus={[Function]}
+          onKeyDown={[Function]}
+          type="text"
+          value=""
+        />
+      </div>
+      <div
+        className="loginContainer__loginDialog__inputLabel"
+      >
+        register.enterAge
+      </div>
+      <div
+        className="loginContainer__loginDialog__input"
+      >
+        <input
+          className="ant-input"
+          onBlur={[Function]}
+          onChange={[Function]}
+          onFocus={[Function]}
+          onKeyDown={[Function]}
+          type="number"
+          value=""
+        />
+      </div>
+      <div
+        className="loginContainer__loginDialog__inputLabel"
+      >
+        register.email
+      </div>
+      <div
+        className="loginContainer__loginDialog__input"
+      >
+        <input
+          className="ant-input"
+          onBlur={[Function]}
+          onChange={[Function]}
+          onFocus={[Function]}
+          onKeyDown={[Function]}
+          type="text"
+          value=""
+        />
+      </div>
+      <div
+        className="loginContainer__loginDialog__inputLabel"
+      >
+        register.password
+      </div>
+      <div
+        className="loginContainer__loginDialog__input"
+      >
+        <input
+          className="ant-input"
+          onBlur={[Function]}
+          onChange={[Function]}
+          onFocus={[Function]}
+          onKeyDown={[Function]}
+          type="password"
+          value=""
+        />
+      </div>
+      <div
+        className="loginContainer__loginDialog__input"
+      >
+        <label
+          className="ant-checkbox-wrapper"
+        >
+          <span
+            className="ant-checkbox"
+            style={Object {}}
+          >
+            <input
+              checked={false}
+              className="ant-checkbox-input"
+              onBlur={[Function]}
+              onChange={[Function]}
+              onFocus={[Function]}
+              type="checkbox"
+            />
+            <span
+              className="ant-checkbox-inner"
+            />
+          </span>
+          <span>
+            register.iAgreeTo
+            <button
+              className="ant-btn ant-btn-link"
+              onClick={[Function]}
+              type="button"
+            >
+              <ul>
+                <li>
+                  r
```

Comment:
fix

Reply:
I had a similar problem with my game. But we couldn't fix it. i18n translation doesn't work with jest. Halit and I tried creating mock but still didn't work. Google wasn't helpful either. Can you tell me a solution?